trivy/trivy-secret.yaml

```diff
@@ -0,0 +1,64 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+# Copyright freiheit.com
+
+# These allow rules are enabled by default. We shall disable all allow rules unless explicitly enabled.
+# The list below consists of all the built-in allow rules. It can be found at [1, 2].
+# 
+# [1] - https://aquasecurity.github.io/trivy/v0.51/docs/scanner/secret/
+# [2] - https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/secret/builtin-allow-rules.go
+disable-allow-rules:
+- "tests"
+- "examples"
+- "vendor"    
+- "usr-dirs"
+- "locale-dir"
+- "markdown"
+- "node.js"
+- "golang"
+- "python"
+- "rubygems"
+- "wordpress"
+- "anaconda-log"
+
+allow-rules:
+  - id: trivy-secret-yaml
+    description: skip this file as it's full of false positives
+    path: trivy/trivy-secret.yaml
+  - id: auth-azure-test-go-1
+    description: skip the JWT in unit test "Kid not present" in file "pkg/auth/azure_test.go" as it was intentionally generated for that test
+    regex: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9\.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ\.WDlNbJFe8ZX6C1mS27xwxg-9tk8vtkk6sDgucRj8xW0'
+  - id: auth-azure-test-go-2
+    description: skip the JWT in unit test "Kid not part of jwks" in file "pkg/auth/azure_test.go" as it was intentionally generated for that test
+    regex: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImFzZGYifQ\.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ\.aNyAK8qpCScGchUmv1q1pBXOddWKN8_7agLUo7pXDog'
+  - id: auth-azure-test-go-3
+    description: skip the RSA private key in function "getToken" in file "pkg/auth/azure_test.go" as it was intentionally generated for unit tests
+    regex: |
+      MIICXQIBAAKBgQC/oyqURHIPNzx4vcKrUUZYr6Bxq2OSD44a63zeIDA1oZkR\+sac
+      tmkub\+8NI49GqrbssWf944v3ZLp8KXMh6i\+U9pkSdDfvKcQUProQ\+Tlm/m0SFXa6
+      h7vq6iVD1uawzN9aQaR7WiKV1TuPGUgE86/l\+XTvLZ/MbKh0tz9j8JtY4QIDAQAB
+      AoGBAICNeROq8oSIfjVUvlDkHXeCoPN/kDS74IzoaYQsPYrMk30/J5qatuYiyk6b
+      CxLRlBIlU\+g5i3vygzKlL4mRqkZuCM4xPbpuW9sdZp61TxWZk7Tm\+SYBTStYSGkT
+      tPmvnKsYWkUh1WDSkeLJqHkRbQXAZJkAKRMYgLu2F29fWOZBAkEA8P31nm/AiDiD
+      dkGSGp4GVQ5BBry3XdP3c6rfzmW8sMElxqoj2watdia72\+grf8eVo8vtsTiOrVUD
+      ZoS5C5GKKQJBAMuSXXQZrBa4qB7YkGi5ysQRQZoegdYZa44q9L9oBE/iEl/ejR1l
+      EKZi\+v2greoIruqczGAD7VbEiwT50\+npH/kCQQDJgpGvOaK0RQ0oBQw2VYzV8mVN
+      TN/HBUcU4PzjiQ6OffMoe3wf2SWSdjD/YNN\+tVTa8dp/Jdun9D4zqydQFRKBAkBV
+      zlPl5AxNZ3g1yELWYbm9\+ygTtlgzznMvcZvIMiffJANqtXv1r\+vctkvlLB0iUJap
+      /X2H2x/nOuD\+L\+/K4KDBAkAHcO3Gv7VZsSHfnd/JfDzxtL0MFWerGZyGlaNFmX27
+      1dWRXvcS5A0zPMgiBWfvHFx2DpSiceffqnis\+UryeE\+L
+  - id: charts-kuberpult-values-yaml
+    description: skip the empty OpenSSH private key as it doesn't expose anything
```

Comment:
I think we should rather adapt the values.yaml to not include this suspicious string.